contracts/VToken.sol

```diff
@@ -255,17 +247,20 @@ contract VToken is Ownable2StepUpgradeable, VTokenInterface, ExponentialNoError,
      * @notice Get a snapshot of the account's balances, and the cached exchange rate
      * @dev This is used by comptroller to more efficiently perform liquidity checks.
      * @param account Address of the account to snapshot
-     * @return (possible error, token balance, borrow balance, exchange rate mantissa)
+     * @return error Always NO_ERROR for compatilibily with Venus core tooling
+     * @return vTokenBalance User's balance of vTokens
+     * @return borrowBalance Amount owed in terms of underlying
+     * @return exchangeRate Stored exchange rate
      */
     function getAccountSnapshot(address account)
         external
         view
         override
         returns (
-            uint256,
-            uint256,
-            uint256,
-            uint256
+            uint256 error,
+            uint256 vTokenBalance,
+            uint256 borrowBalance,
+            uint256 exchangeRate
```

Comment:
Why do we still return an error here?

Reply:
Because there are a lot of external tooling around Compound and Venus that rely on getHypotheticalAccountLiquidity or getAccountSnapshot – subgraphs, liquidation monitors, dashboards and whatnot. I'd rather keep the interfaces compatible. And there was a specific request from Simeon to always return NO_ERROR from mint/repay/etc. – otherwise our swap contract will need to somehow detect if vToken belongs to IL or core.